target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,6 +138,8 @@ case "$FIRMWARE" in
 	archer-c59-v1|\
 	archer-c60-v1)
 		ath10kcal_extract "art" 20480 12064
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
Have you tried contacting kvalo/QCA via the Mailing-lists: [linux-wireless](https://wireless.wiki.kernel.org/en/developers/mailinglists#stats_and_requirements_of_linux-wireless_mailing_list) and [ath10k-devel](http://lists.infradead.org/mailman/listinfo/ath10k )?
As I didn't see anything there about the QCA9886 there.